.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+<!-- What does this PR do? Please list any issue that these changes address and how you have gone about implementing the changes -->
+
+## Checklist
+
+I confirm that I have completed the following checks:
+
+**Testing**
+- [ ] If I have changed any of the Python or Fortran code, I have rebuilt and run the tests locally and they have passed.
+- [ ] I have addressed any differences in the regression tests caused by this pull request in the comments.
+- [ ] I have added new tests where appropriate for the changes I have made.
+- [ ] If I have had to change any existing unit or integration tests, I have justified this change in the pull request comments.
+
+**Quality**
+- [ ] I have ensured my changes are formatted correctly and conform to PROCESS' quality standards.
+- [ ] If I have made documentation changes, I have checked they render correctly.
+
+**Miscellaneous**
+- [ ] I have created a changelog entry for my change, if appropriate.
+- [ ] I have added documentation for my change, if appropriate.
+- [ ] **I confirm my contribution is made to PROCESS under an MIT license**.
```

Comment:
Is this really necessary for every PR? If you're contributing to an MIT-licenced code, isn't this already clearly implied?

Reply:
This is true, #2892 will make this clear in the contributing guide, so we probably don't need this